LandsitesTools/prepare_nlp_cases.py

```diff
@@ -1 +1,282 @@
+        cases_df.loc[idx,"res"] = cases_dict.get(case).get("res")
+        cases_df.loc[idx,"lat"] = cases_dict.get(case).get("lat")
+        cases_df.loc[idx,"lon"] = cases_dict.get(case).get("lon")
+        
+    print('---------------------------------------------------------------')
+    
+    
+    ##### Get user input #####
+    # Variable to evaluate user input
+    build_in = "o"
+    ### Repeat until choice is correct
+    while(build_in == "o"):
+        
+        case_indices_str = \
+        input("Enter the indices of the cases to build (seperated by space or comma if more than one): ")
+
+        ### Turn into list, remove duplicates, check for valid input
+        try:
+
+            case_indices_int = [int(idx) for idx in re.split('[ ,;]+', case_indices_str)]
+            case_indices_int = sorted(list(set(case_indices_int)))
+
+            ### Any value out of array range?
+            if max(case_indices_int) >= cases_df.shape[0] or min(case_indices_int) < 0:
+                raise ValueError("At least one index out of range.")
+
+        except ValueError:
+            print("")
+            print("Please only enter integers within index range!")
+            print("")
+            raise
+
+        ### Make sure input is correct
+        build_in = ""
+        while build_in not in ["y", "o", "a"]:
+            build_in = input("Prepare the following cases: " \
+                             + ", ".join([cases_df.loc[idx,"name"] for idx in case_indices_int]) \
+                             + "? ([y]es/[o]ther/[a]bort)")
+    ### Exit if entered
+    if build_in == "a":
+        sys.exit("Exit by user.")
+        
+    ### Store names of cases
+    cases_to_build = cases_df.loc[case_indices_int,"name"]
+    
+    
+    ################################### Machine configs ###################################
+    
+    ### Should also be stored in an external file!
+    machine_cfgs_df = pd.DataFrame(columns=["name","machine","compset","project"])
+    saga = pd.Series(data={"name":"SAGA", 
+                           "machine":"saga",
+                           "compset":"2000_DATM%1PTGSWP3_CLM50%FATES_SICE_SOCN_MOSART_SGLC_SWAV", 
+                           "project":"nn2806k"}, name="saga")
+    fram = pd.Series(data={"name":"Fram",
+                           "machine":"fram",
+                           "compset":"2000_DATM%1PTGSWP3_CLM50%FATES_SICE_SOCN_MOSART_SGLC_SWAV", 
+                           "project":"nn2806k"}, name="fram")
+    galaxy = pd.Series(data={"name":"GALAXY",
+                             "machine":"espresso",
+                             "compset":"2000_DATM%1PTGSWP3_CLM50%FATES_SICE_SOCN_MOSART_SGLC_SWAV", 
+                             "project":""}, name="galaxy")
+
+    machine_cfgs_df = machine_cfgs_df.append(saga, ignore_index=True)
+    machine_cfgs_df = machine_cfgs_df.append(fram, ignore_index=True)
+    machine_cfgs_df = machine_cfgs_df.append(galaxy, ignore_index=True)
+
```

Comment:
Because we did not discuss how this information should be stored yet, quick fix to test general functionality (hence the comment above) :) any suggestions?